controllers/kogitoserverlessbuild_controller_test.go

```diff
@@ -37,6 +37,7 @@ func TestKogitoServerlessBuildController(t *testing.T) {
 		WithRuntimeObjects(ksb, ksw).
 		WithRuntimeObjects(test.GetKogitoServerlessPlatformInReadyPhase("../config/samples/"+test.KogitoServerlessPlatformWithCacheYamlCR, namespace)).
 		WithRuntimeObjects(test.GetKogitoServerlessOperatorBuilderConfig("../", namespace)).
+		WithStatusSubresource(ksb, ksw).
```

Comment:
Why `WithStatusSubresource` isntead of `WithStatusSubResource`

Reply:
This is a signature of the controller-runtime package: https://github.com/kubernetes-sigs/controller-runtime/pull/2259